app/enterprise/1.3-x/deployment/installation/amazon-linux.md

```diff
@@ -1,321 +1,312 @@
+    ```
 
-## Step 6. Verify Kong is Receiving Requests
+4. Verify Kong Enterprise is working:
 
-##### Install HTTPie to easily send requests to Kong *(optional)*
-  ```
-  sudo yum install python36
+    ```bash
+    $ curl -i -X GET --url http://localhost:8001/services
+    ```
+    
+5. You should receive a `HTTP/1.1 200 OK` message.
 
-  sudo pip-3.6 install --upgrade pip setuptools
+## Step 6. Finalize Configuration and Verify Installation
 
-  sudo /usr/local/bin/pip3.6 install --upgrade httpie
-  ```
+### Enable and Configure Kong Manager
 
-#### Test your connection to Kong
+1. To access Kong Enterprise's Graphical User Interface, Kong Manager, update the `admin_gui_url` property in `/etc/kong/kong.conf` file the to the DNS, or IP address, of the CentOS system. For example:
 
-**httpie**
-  ```
-  http :8001/services 
+    ```
+    admin_gui_url = http://<DNSorIP>:8002
+    ```
   
-  if RBAC is enabled:
-  *http :8001/services Kong-Admin-Token:$KONG_PASSWORD
-  ```
-
-**curl**
-  ```
-  curl -i -X GET --url http://localhost:8001/services
-
-  if RBAC is enabled:
-  curl -i -X GET --url http://localhost:8001/services --header 'kong-admin-token:<KONG_PASSWORD>'
-  ```
-
-
-You should receive a `200 OK` response:
-
-```
-HTTP/1.1 200 OK
-Access-Control-Allow-Origin: *
-Connection: keep-alive
-Content-Length: 23
-Content-Type: application/json; charset=utf-8
-Date:
-Server: kong/kong-enterprise-edition
-X-Kong-Admin-Request-ID: 
-
-{
-    "data": [],
-    "next": null
-}
-```
-
-If you did not receive a `200` check out the troubleshooting section below or contact your Sales or Support Representative for assistance.
-
-
-## Troubleshooting
+    This setting needs to resolve to a network path that will reach the Amazon Linux host.
+  
+2. It is necessary to update the administration API setting to listen on the needed network interfaces on the CentOS host. A setting of `0.0.0.0:8001` will listen on port `8001` on all available network interfaces.
+  
+    ```
+    admin_listen = 0.0.0.0:8001, 0.0.0.0:8444 ssl
+    ```
+  
+3. You may also list network interfaces separately as in this example:
+  
+    ```
+    admin_listen = 0.0.0.0:8001, 0.0.0.0:8444 ssl, 127.0.0.1:8001, 127.0.0.1:8444 ssl
+    ```
+  
+4. Restart Kong for the setting to take effect:
 
-##### I receive an error when downloading the RPM Package
-  Ensure that you are using your Bintray Username and Bintray account **password **in the `wget` request - note that this is different from your Bintray API KEY.
+    ```bash
+    $ sudo /usr/local/bin/kong restart
+    ```
+  
+5. You may now access Kong Manager on port `8002`.
 
+### Enable the Developer Portal
 
-##### HTTP 401 - Unauthorized
-  If you have enabled RBAC all request must be sent with the `kong-admin-token` header 
+1. Kong Enterprise's **Developer Portal** can be connected by setting the `portal` property to `on` and setting the `portal_gui_host` property to the EC2 instance's `IPv4` address
```

Comment:
Are we using "EC2"?

Reply:
Yes, EC2 is the compute service running Amazon Linux.